Subject: Add lua support to ptxdist 1.99: add autogen.sh
From: Erwin Rol <erwin@erwinrol.com>

This patch add lua support to ptxdist 1.99. It has a menu entry under
scripting languages. And a patch that makes it possible to use autoconf
to build.

This part of the patch adds an autogen.sh script.

Signed-off-by: Erwin Rol <erwin@erwinrol.com>
Signed-off-by: Robert Schwebel <r.schwebel@pengutronix.de>

---
 autogen.sh |    5 +++++
 1 file changed, 5 insertions(+)

Index: lua-5.1.3/autogen.sh
===================================================================
--- /dev/null
+++ lua-5.1.3/autogen.sh
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+autoreconf -fvi
+
+echo "Now run ./configure"
